### PR TITLE
Add donate slack notification

### PR DESCRIPTION
**Pull request by denys-medynskyi**
🍅 

### Description

When a user donates money through https://pivorak.com/donate we should send Slack notification to encourage others to donate.

This PR adds route `/donate/webhook` where WayForPay service is sending data.
WayForPay webhook URL can be configured here - https://m.wayforpay.com/en/mportal/payed-block-configure/notice-update?id=50029
You need to ask Anna, Sofia or Denys to access there.

### Screenshot

Describe how to test your changes if specs are not covering completely.
Screen Shot 2019-12-08 at 7 27 08 AM

### How to test

```
curl --request POST 'http://localhost:3000/donate/webhook' \
--header 'Content-Length:593' \
--header 'X-Forwarded-Proto:https' \
--header 'X-Forwarded-For:88.85.72.203' \
--header 'Connection:close' \
--header 'Accept:*/*' \
--header 'X-Request-Start:t=1575791662.136' \
--header 'Host:pivorak.com' \
--header 'Version:HTTP/1.0' \
--header 'Content-Type:application/x-www-form-urlencoded' \
--header 'X-Forwarded-Ssl:on' \
--data '{"merchantAccount":"pivorak_com","orderReference":"WFP-2102129-5decabc01799e","merchantSignature":"8c3a5a8211b9556d475be7a1f5f5b5d3","amount":5,"currency":"UAH","authCode":"855512","email":"denys.medynskyi@gmail.com","phone":"380637589464","createdDate":1575791552,"processingDate":1575791590,"cardPan":"53****6819","cardType":"MasterCard","issuerBankCountry":"Ukraine","issuerBankName":"MONObank","recToken":"","transactionStatus":"Approved","reason":"Ok","reasonCode":1100,"fee":0.13,"paymentSystem":"card","acquirerBankName":"WayForPay","cardProduct":"debit","clientName":"DENYS MEDYNSKYI"}=None'
```

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -275,3 +275,8 @@ en:
     emails:
       label: 'Emails separated by comma. Example: john@m.com OR john@m.com, peter@m.com'
       error: "Should be comma separate list. Following emails are invalid: %{invalid_emails}"
+
+  slack:
+    new_donation_notification_message: |-
+      :pivorak: Щойно задонатили %{amount} %{currency} :tada:.
+      Дякую :bow: тобі анонімний #pivorak-ер!
```
Comment (sveredyuk) [inline config/locales/en.yml:282]: Дякую :bow: тобі анонімний #pivorak-er!

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+RECAPTCHA_SITE_KEY=SOME_RECAPTCHA_SITE_KEY
```
Comment (denys-medynskyi) [inline .env.example]: Remove

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+RECAPTCHA_SECRET_KEY=SOME_RECAPTCHA_SECRET_KEY
```
Comment (denys-medynskyi) [inline .env.example]: Remove

File: Gemfile.lock [hunk 4]
```diff
@@ -712,4 +716,4 @@ RUBY VERSION
    ruby 2.3.1p112
 
 BUNDLED WITH
-   1.16.6
+   1.17.3
```
Comment (denys-medynskyi) [inline Gemfile.lock]: Do not update

File: config/locales/en.yml (hunk 1 above)
Comment (skrix) [reply config/locales/en.yml:282]: Main language Ukrainian, so the ending of brand names when it should be modified according to rules should be also in Ukrainian.

**Review by skrix — APPROVED** [review]
Almost OK

**Comment by skrix** [comment]
Good work, let's migrate the server and deploy this feature asap! ;)

**Pull request merged**